docs/testing.md

```diff
@@ -53,19 +53,34 @@ These steps can be acheived using commands below.  Look at the [e2e helper
 file](../hack/e2e/run-rp-and-e2e.sh) to understand each of the bash functions
 below.
 
-### Run a specific test
+## Smoke tests
 
-End to end tests are run using ginkgo. You can run subsets of tests or ignore some tests by following the [ginkgo documentation](https://onsi.github.io/ginkgo/#filtering-specs)
+We have some other tests under `test/e2e` that are not part of the CI.
+These tests are labelled as `smoke` and are supposed to be run to check the basic functionality of the OCP cluster.
+They can be used as a gap analysis for new OCP versions or the installer updates.
+
+You can run the smoke tests by running the following command:
+
+```bash
+E2E_LABEL=smoke make test-e2e
+```
+
+If you want to run both e2e and smoke tests:
+
+```bash
+E2E_LABEL= make test-e2e
```

Comment:
Is this required, or can we simplify?
```suggestion
make test-e2e
```

Reply:
It's required to disable default.

https://github.com/Azure/ARO-RP/blob/50939e1a381c0795bc13d2bd3d32c48ca39cfa40/Makefile#L6